Player/Player.gd

```diff
@@ -87,6 +94,8 @@ const MAX_SLOPE_ANGLE: int = 40
 
 var is_crouching: bool = false
 var stamina: float = 100.0
+var xVal = 0
```

Comment:
This variable name is unclear. My understanding is that this variable stores the last y value that the player had while they were on the ground.